github/create-github-release/example/run.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: create-github-release-pipeline
+spec:
+  workspaces:
+    - name: shared-workspace
+    - name: upload
+    - name: release-notes
+  tasks:
+    - name: fetch-repository
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+      params:
+        - name: url
+          value: https://github.com/divyansh42/aws-cli.git
+        - name: subdirectory
+          value: ""
+        - name: deleteExisting
+          value: "true"
+    - name: create-github-release
+      taskRef:
+        name: create-github-release
+      params:
+        - name: TAG
+          value: "v1.0"
+        - name: REVISION
+          value: master
+        - name: RELEASE_FILE_NAME
+          value: release.md
+        - name: GITHUB_TOKEN_SECRET
+          value: github-token
+        - name: GITHUB_TOKEN_SECRET_KEY
+          value: GITHUB_TOKEN
+      runAfter:
+        - fetch-repository
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+        - name: input
+          workspace: upload
+        - name: release-notes
+          workspace: release-notes
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: create-github-release-pipeline-run
+spec:
+  pipelineRef:
+    name: create-github-release-pipeline
+  workspaces:
+    - name: shared-workspace
+      persistentVolumeClaim:
```

Comment:
you could use a `volumeClaimTemplate` here, which would mean you could drop the separate PVC YAML file if you wanted.

This workspace would become:

```
- name: shared-workspace
  volumeClaimTemplate:
    spec:
      accessModes:
      - ReadWriteOnce
      resources:
        requests:
          storage: 500Mi 
```

The nice benefit is that someone using this example gets the PVCs cleaned up for free when they delete the PipelineRun.

Reply:
Okay, thanks for suggestion. This is really helpful :slightly_smiling_face: